every_election/apps/organisations/boundaries/boundary_bot/scraper.py

```diff
@@ -161,17 +164,19 @@ def get_org_from_reg_code(self, register_code):
         return org
 
     def get_review_from_db(self, record):
-        if record["legislation_title"]:
+        if record["legislation_url"]:
+            legislation_year = self.get_legislation_year(record)
+
             try:
                 result = OrganisationBoundaryReview.objects.filter(
-                    legislation_title=record["legislation_title"],
+                    legislation_url__contains=f"/{legislation_year}/",
                     slug=record["slug"],
                 )
                 if len(result) == 1:
                     return result
                 if len(result) > 1:
                     raise ScraperException(
-                        f"More than one review found with same legislation_title: {record['legislation_title']}"
+                        f"More than one review found for {record['legislation_title']} with same year: \n {str(record)}",
```

Comment:
The duplicate here is on year and slug, so lets say that in the error.

```suggestion
                        f"More than one review found for {record['slug']} with year {legislation_year}",
```

I don't think we need to dump the whole record